<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>QmitkXnatProjectWidgetControls</class>
  <widget class="QWidget" name="QmitkXnatProjectWidgetControls">
    <property name="enabled">
      <bool>true</bool>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>350</width>
        <height>154</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Qmitk Template</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
        <widget class="QLabel" name="descriptionLabel">
          <property name="text">
            <string>Description:</string>
          </property>
        </widget>
      </item>
      <item row="2" column="0">
        <widget class="QLabel" name="nameLabel">
          <property name="text">
            <string>Name:</string>
          </property>
        </widget>
      </item>
      <item row="4" column="0">
        <widget class="QLabel" name="piLabel">
          <property name="text">
            <string>PI:</string>
          </property>
        </widget>
      </item>
      <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="columnLabel">
          <property name="font">
            <font>
              <weight>75</weight>
              <bold>true</bold>
            </font>
          </property>
          <property name="text">
            <string>Project Information</string>
          </property>
          <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
        </widget>
      </item>
      <item row="3" column="1">
        <widget class="QLineEdit" name="descriptionLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="2" column="1">
        <widget class="QLineEdit" name="nameLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="1" column="0">
        <widget class="QLabel" name="idLabel">
          <property name="text">
            <string>ID*:</string>
          </property>
        </widget>
      </item>
      <item row="4" column="1">
        <widget class="QLineEdit" name="piLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="1" column="1">
        <widget class="QLineEdit" name="idLineEdit">
          <property name="readOnly">
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item row="5" column="0" colspan="2">
        <widget class="QLabel" name="mandatoryLabel">
          <property name="text">
            <string>* mandatory field</string>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources />
  <connections />
</ui>
